app.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+# -*- coding:utf-8 -*-
+
+from flask import Flask, request, jsonify, send_from_directory, redirect # , send_file
+from flask_cors import CORS
+import pdf
```

Comment:
The current version totally OK, since it's an initial version and work in progress..  However, an encapsulation as a Python module would be nice. :-)